src/pages/home-page.js

```diff
@@ -185,7 +153,7 @@ const HomePage = (props) => {
 
       {/* Code, name and desc hardcoded for testing purposes */}
       <Grid columns={3}>
-        {buildGrid()}
+        <CardGrid courses={courses}></CardGrid>
```

Comment:
you don't pass any children (in between `>` and `</` so we can make this a self closing tag

`<CardGrid courses={courses} />` 

Otherwise this looks really good!